odoo/addons/base/tests/data.py

```diff
@@ -0,0 +1,37 @@
+from base64 import b64decode
```

Comment:
Since we're making a new file. Please make as many file as necessary, and `open` them.

```py
with file_open('base/tests/image.png') as _png_file:
    PNG_RAW = _png_file.read()
    PNG_B64 = b64encode(PNG_RAW)

...
```

Reply:
They could also be stored as attachment in XML data.